proto/buf/alpha/registry/v1alpha1/list.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020-2021 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+message ListOptions {
```

Comment:
This is sort of an anti-pattern in Protobuf - we generally want to expand this for each RPC. Combining this makes the API more difficult for end-users to deal with, and generally only makes sense from an implementation perspective (which is the implementer's problem, not the client's).

Reply:
Interesting, I didn't know this was an anti-pattern. 

I like the implementor vs consumer point, but being able to logically group options potentially makes it easier for the consumer (in addition to the implementor)? Both sides get consistent behaviour across all List endpoints.

Reply:
You can have consistent behavior without having to wrap these fields in another message